src/Login.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react'
+import { Alert, Card, Form, Button } from "react-bootstrap"
+import { Navigate } from 'react-router-dom'
+import { setLoggedIn, isLoggedIn } from './App'
+import "./App.css"
+import "./Login.css"
+
+
+function CredentialsAlert(state) {
+    if (state.props.state.alert === "loggingIn") {
+        return (
+            <Alert variant="success">
+                Logging in!
+            </Alert>
+        );
+    } else if (state.props.state.alert === "invalid") {
+        return (
+            <Alert variant="danger">
+                Invalid email or password.
+            </Alert>
+        );
+    }
+}
+
+class Login extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            email: "",
+            password: "",
+            alert: ""
```

Comment:
I think this may be clearer as a boolean (true/false) value where we can check if the credentials are valid or not. It just helps to avoid string matching where the person passing in the state value has to know that the CredentialsAlert is looking for "loggingIn" or "invalid", typos, etc.